### PR TITLE
Productive label for Workflow

### PR DESCRIPTION
**Pull request by artemkovalyov**
## What Has Changed?

Add a note about productive use to the header.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
~~- [] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.~~

### CODE REVIEW

File: docs/js/features/openapi/pregenerated-clients/workflow.mdx [hunk 1]
```diff
@@ -14,6 +14,12 @@ keywords:
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::tip Generally Available
+
+This library is released for the **productive use**.
```
Comment (marikaner) [inline docs/js/features/openapi/pregenerated-clients/workflow.mdx:19]:
```suggestion
This library is released for **productive use**.
```

**Review by marikaner — COMMENTED** [review]
I am wondering whether this message is helpful. All libraries that we provide are ready for productive use, unless noted otherwise. This might open the question whether other things are not recommended for productive use, if we don't state it explicitly.

What are you hoping to achieve with this? Maybe we can achieve it otherwise?

**Comment by artemkovalyov** [comment]
Explicit is better than implicit. Customers asked for this. A couple of times Beta annotation wasn't removed in API docs which lead to confusion.  Also, the SAP Workflow was in Beta for more than half a year in Java for example. Now same might be true for Business Rules. Please, suggest how we can achieve this otherwise, or we can continue using this.

**Comment by artemkovalyov** [comment]
@marikaner, have you had a chance to think about another way to achieve it? Do we need more discussion about it?

**Pull request closed without merging**